### ISSUE TITLE
[Bug] 1.1.0-pre1 Popup binding does not work on iOS

### ISSUE THREAD

**Issue by tranb3r** [xamarin/XamarinCommunityToolkit#999]
### Description

In XCT 1.1.0-pre1, binding in a popup does not work on iOS (but it does work on Android).

### Steps to Reproduce

1. Open sample app on iOS
2. Go to the XamlBindingPopup demo
3. Open the popup

### Expected Behavior

The popup should contain a title and a message.

### Actual Behavior

On iOS, the popup is empty. The bindings do not work.

### Basic Information

- Version with issue: 1.1.0-pre1
- IDE: VS 2019 16.8.6
- iOS:  14.4

### Reproduction imagery

![xct](https://user-images.githubusercontent.com/12516566/109552452-16d96980-7ad2-11eb-9ecd-dc65fe8cc703.png)

**Comment by pictos** [comment]
@tranb3r can you please provide a repro?

**Comment by geskill** [comment]
Can confirm. Until a proper fix is available you can use this as workaround:

Remove BindingContext from popup, add to inner root view element.

e.g. **XamlBindingPopup.xaml**
```xml
<StackLayout Style="{StaticResource PopupLayout}">
    <StackLayout.BindingContext>
        <vm:XamlBindingPopupViewModel />
    </StackLayout.BindingContext>
    ...
</StackLayout>
```

**Comment by pictos** [comment]
cc: @ahoefling

**Comment by AndreiMisiukevich** [comment]
Fixed in #1003
